sql/sqlConstraintsUpdate.sql

```diff
@@ -0,0 +1,21 @@
+--Update Script
+
+SELECT *
```

Comment:
I think we can remove this SELECT * FROM Director and leave the alter table steps.